libirods_rule_engine_plugin-audit_amqp.cpp

```diff
@@ -510,10 +506,7 @@ namespace
 			std::uint64_t time_ms = ts_clock::now().time_since_epoch() / std::chrono::milliseconds(1);
 			json_obj["@timestamp"] = time_ms;
 
-			char host_name[MAX_NAME_LEN];
-			gethostname(host_name, MAX_NAME_LEN);
-			insert_or_parse_as_bin(json_obj, "hostname", host_name, time_ms);
```

Comment:
Passing hostnames through `insert_or_parse_as_bin` was introduced in the qpid-cpp refactor commit. Instead of removing these calls here, just don't introduce them in the first place.